### DISCUSSION TITLE
How about making the overlays behave similar to the GeoJSON?

### DISCUSSION THREAD

**Discussion by Moonbase59** [javalent/obsidian-leaflet#118] [Ideas]
For added usability, and more similar behaviour: Since we’ve now tackled the more complicated case first … how about making the circle overlays behave the same?

Notably, …

- when left-clicking, move their tooltip "here" (because one is often _within_ a larger overlay), and
- allowing Ctrl-click on an overlay to also auto-zoom and auto-center to it (like with GeoJSON)?

(Sorry, couldn’t sleep and was dreaming about maps, layers and stuff …)

**Comment by valentine195** [comment]
Both of these behaviors are now present in 3.18.9.
